### ISSUE TITLE
Limit suggestions to the current manuscript

### ISSUE THREAD

**Issue by jeromepl** [DDMAL/cantus#274]
Most of the code is already in place for this, as Solr has a "context field" in suggesters that allows exactly this. However, I ran into some problems with it while upgrading to Solr 6 so I decided to leave that for another time. This does not break the suggesters, but suggestions from things found in other manuscripts are shown.

**Comment by jeromepl** [comment]
See commit 0b10e8026e3ed0c0e4d60c3fc591cc3e05be0678

**Comment by jeromepl** [comment]
I have attempted to fix this in many different ways, none of which worked. Solr's context field for suggesters has almost no documentation online and it is thus very hard to find examples of it working.
For documentation purposes, here is what I have tried:
- Using a field of type solr.TextField as context.
- Using a field of type solr.TextField as context, and specifying to use the KeywordTokenizer for indexing and querying (tried using a copy field and an entirely new field).
- When performing a query, sending the context query string between quotes, parentheses and nothing. In the case of the quotation marks, I got an error (see below).

All give me the same behavior: If I ask for a suggestion without specifying the context, it returns all matching suggestions (this works as intended). If I specify the context, I get no suggestions back. Finally, if I put the context inside quotation marks, I get an error saying `java.lang.IllegalStateException: field "contexts" was indexed without position data; cannot run PhraseQuery (phrase=contexts:"CDN Hsmu M2149 L4")`. I believe that the context field is tokenized and indexed differently inside Solr before being sent to the suggestion module of Lucene.
In the example above, the original context I sent was `CDN-Hsmu M2149.L4` and became `CDN Hsmu M2149 L4` in the error message, even when I made the field in question use the KeywordTokenizer which takes the whole string as one token.
